packages/survey-core/src/survey-element.ts

```diff
@@ -850,7 +850,7 @@ export class SurveyElement<E = any> extends SurveyElementCore implements ISurvey
   }
 
   public get isDefaultV2Theme() {
-    return this.survey && this.survey.getCss().root == "sd-root-modern";
+    return this.survey && this.survey.getCss().root.includes("sd-root-modern");
```

Comment:
I'd reccomended to use indexOf here because includes willn't work in IE

Reply:
Where's the list of supported browsers on surveyjs.io? I'm very skeptical that the library works properly on IE 11 and I dont see it as a pro if it does. IE 11 is not supported by Microsoft on any platform; not sure why you'd want to waste your time on it.

Reply:
Fixed it, under protest!